### PR TITLE
fix: use akuitybot PAT for backport worfklow

### PR DESCRIPTION
**Pull request by hiddeco**
This should ensure workflows are properly triggered in all scenarios. Follow up on #1936.

### CODE REVIEW

File: .github/workflows/backport.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
         # xref: https://github.com/korthout/backport-action#inputs
         with:
           # Use token to allow workflows to be triggered for the created PR
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+          github_token: ${{ secrets.AKUITYBOT_PAT }}
```
Comment (krancour) [inline .github/workflows/backport.yaml:25]: Would `permissions.pull-requests: write` possibly work?

https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

Comment (krancour) [reply .github/workflows/backport.yaml:25]: Oh... nevermind... I was looking at perms at the top of the file. I see this job already has the one I mentioned.

Comment (krancour) [reply .github/workflows/backport.yaml:25]: Oh... this needs to be enabled at the org _and_ repo level. Repo settings are already correct. Perhaps @kencochrane can check if `Allow GitHub Actions to create and approve pull requests` is checked at the org level.

Comment (krancour) [reply .github/workflows/backport.yaml:25]: @kencochrane were the org settings already like that or did you check that box just now?

Comment (kencochrane) [reply .github/workflows/backport.yaml:25]: I didn't change any values; that is what it had.

Comment (krancour) [reply .github/workflows/backport.yaml:25]: Thanks

**Review by krancour — APPROVED** [review]
I am dumbfounded by why the `pull-requests: write` permission isn't working, but am willing to accept this workaround in the interim.

File: .github/workflows/backport.yaml (hunk 1 above)
Comment (hiddeco) [reply .github/workflows/backport.yaml:25]: @kencochrane I have a feeling the repository needs to be added to the allow list of the organization global secret.

Comment (kencochrane) [reply .github/workflows/backport.yaml:25]: we don't have an `AKUITYBOT_PAT` org secret on the `akuity` GitHub organization; we have one on our `akuityio` organization. That is probably the issue.

Comment (kencochrane) [reply .github/workflows/backport.yaml:25]: We will have to ask @jessesuen if he can add it since I don't have the AKUITYBOT_PAT value to add it.

**Pull request merged**